Makefile

```diff
@@ -12,6 +12,9 @@ DEFAULT_ARTIFACTS_CONTAINER_PORT = 8081
 MAKE_HELPERS_PATH = $(shell pwd)/hack/make-helpers
 SUPPORTED_VERSIONS_JSON = $(shell pwd)/supported-versions.json
 
+# Fetching Image Builder commit ID from supported-versions.json
+IMAGE_BUILDER_COMMIT_ID:=$(shell jq -r '."${KUBERNETES_VERSION}".extra_params.image_builder_commit' "${SUPPORTED_VERSIONS_JSON}")
```

Comment:
You can move this to [run-artifacts-container.sh](https://github.com/vmware-tanzu/vsphere-tanzu-kubernetes-grid-image-builder/blob/main/hack/make-helpers/run-artifacts-container.sh)

Reply:
My bad this should have been [build-node-image.sh](https://github.com/vmware-tanzu/vsphere-tanzu-kubernetes-grid-image-builder/blob/main/hack/make-helpers/build-node-image.sh) instead of [run-artifacts-container.sh](https://github.com/vmware-tanzu/vsphere-tanzu-kubernetes-grid-image-builder/blob/main/hack/make-helpers/run-artifacts-container.sh)